libsolidity/ast/AST_accept.h

```diff
@@ -67,6 +67,18 @@ void ImportDirective::accept(ASTConstVisitor& _visitor) const
 	_visitor.endVisit(*this);
 }
 
```

Comment:
This needs changes in all ast nodes that inherit from 'FormallyDocumented'

Reply:
So that would be
```
if (m_documentation)
    m_documentation->accept(_visitor);
```
Or am I missing something else?

Reply:
Yes!